pkgs/development/python-modules/bitmath/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, buildPythonPackage, fetchPypi, isPy36, progressbar, mock }:
+
+buildPythonPackage rec {
+  pname = "bitmath";
+  version = "1.3.1.2";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1k8d1wmxqjc8cqzaixpxf45k6dl1kqhblr0g4wyjl0qa18q8wasd";
+  };
+
+  doCheck = ! isPy36; # progressbar is incompatible with Py3?
+  checkInputs = [ progressbar mock ];
+
+  meta = with stdenv.lib; {
+    description = ''
+      Module for representing and manipulating file sizes with different prefix
+      notations
+    '';
```

Comment:
Please use `"` and a single line.

Reply:
The line is 94 characters long if I stuff it all onto a single line; is that okay?

Reply:
It's okay for me. I don't think there's a general limit of 80 characters for nixpkgs.